bundles/ranvier-areas/areas/limbo/items.yml

```diff
@@ -10,6 +10,9 @@
   roomDesc: "A moldy slice of cheese"
   keywords: [ "slice", "cheese", "moldy" ]
   description: "A yellow, slightly moldy slice of cheese. Only a rat could find this appetizing."
+  behaviors:
```

Comment:
The way this currently works would break the fetch quest. Fetch quest listens for the 'drop' event so if decay bypasses `drop` by directly removing the item the drop event won't get fired so the quest won't be updated with the fact that the player no longer has the item